.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+    branches:
+      - master
+jobs:
+  test:
+    name: Run tests and publish test coverage
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x, 18.x, 19,x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/    
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'  
+      - name: Install dependencies
+        run: npm install
+      - name: Run tests
+        run: npm test 
+      - name: Run coverage
+        run: npm run coverage 
+      - name: Upload coverage to Codecov
```

Comment:
its pushing coverage for all the version which is not required instead can we add a if condition for the version that we releasing which is 16

Reply:
added condition above code coverage command , so it will check if the node version is '16.x', it will trigger